_locales/en/messages.json

```diff
@@ -1,4 +1,12 @@
 {
+    "menuLabels": {
+      "file": "&File",
```

Comment:
Sadly, you'll need to split this into a number of different top-level keys. Our localization system requires that you have a top level name key (which is what our `i18n` function in the app uses to find strings), then the actual localizable content is under its `message` key. We have to do it that way because our translation systems rely on that structure.